cleanlab/object_detection/summary.py

```diff
@@ -144,7 +144,7 @@ def visualize(
         allowed_image_formats = set(["png", "pdf", "ps", "eps", "svg"])
         image_format = (
             save_path[-3:]
-            if len(save_path) > 2
+            if len(save_path) > 3
```

Comment:
Wouldn't this still not work for `save_path = "image.ps"` ? In this case `save_path[-4] == "e"` and `save_path[-3:] == ".ps"`